Mathlib/CategoryTheory/Limits/Indization/Category.lean

```diff
@@ -286,4 +289,17 @@ theorem Ind.exists_nonempty_arrow_mk_iso_ind_lim {A B : Ind C} {f : A ⟶ B} :
   · simpa using
       (P.parallelPairIsoParallelPairCompIndYoneda.hom.naturality WalkingParallelPairHom.left).symm
 
+section Small
+
+variable (C : Type u) [Category.{u} C] [HasFiniteColimits C]
+
+
+/-- For small categories `C : Type u`, the category of Ind-objects `Ind C` is equivalent to the
```

Comment:
Again here you should probably mention that `C` needs to be finitely cocomplete.